index.html

```diff
@@ -0,0 +1,1383 @@
+                                </h6>
+                                <p class="small"><strong>BY ASAD M. KHAN</strong></p>
+                            </div>
+                        </article>
+                        <article class="d-flex border border-top-0 border-left-0 border-right-0 border-secondary pb-2">
+                            <div class="mt-3"><img src="https://d.newsweek.com/en/full/1598068/kelsey-baker.png?w=63&h=63&f=43880c72e8f62d989abb32aba0d5392f" alt="hello"/></div>
+                            <div class="ml-3 mt-3">
+                                <h6>
+                                    <a href="https://www.newsweek.com/military-does-try-prevent-sexual-violence-its-just-too-late-opinion-1518731" class="text-dark">
+                                        <strong>The Military Does Try to Prevent Sexual Violence. It's Just Too Late</strong>
+                                    </a>
+                                </h6>
+                                <p class="small"><strong>BY KELSEY BAKER</strong></p>
+                            </div>
+                        </article>
+                        <article class="d-flex mb-4">
+                            <div class="mt-3"><img src="https://d.newsweek.com/en/full/1610783/judith-derrick.png?w=63&h=63&f=ed50fb7d3a8e51c8ac2f5f33d33612ea" alt="hello" /></div>
+                            <div class="ml-3 mt-3">
+                                <h6>
+                                    <a href="https://www.newsweek.com/black-lives-matter-money-budgets-politicians-bronx-1518722" class="text-dark"><strong>Politicians Who Say Black Lives Matter: Put Your Budget Where Your Mouth Is</strong></a>
+                                </h6>
+                                <p class="small"><strong>BY JUDITH K. DIMON AND DERRICK H. LEWIS</strong></p>
+                            </div>
+                        </article>
+                    </div>
+                    <div class="d-flex d-md-block my-4 bg-light justify-content-center">
+                        <div class="flex-column justify-content-center my-4 p-3 email-subscription-top h-auto m-1">
+                            <div class="d-flex m-2">
+                                <div class="mt-2"><img src="https://g.newsweek.com/themes/newsweek/favicons/favicon-32x32.png" alt="hello" /></div>
+                                <span class="ml-1"><strong>START YOUR DAY WITH OUR TOP 5 ARTICLES</strong></span>
+                            </div>
+                            <div class="ml-2 mr-2 mt-3">
+                                <input type="text" class="form-text form-control" placeholder="Email address" />
+                                <input type="submit" class="form-submit btn btn-primary btn-block btn-danger mt-3" value="FREE SIGN UP" />
+                            </div>
+                        </div>
+                    </div>
+                    <div class="row mt-3 right-news-cont d-none d-lg-block">
```

Comment:
- [ ] You are not using right-news-cont in your css anymore, please take it out

Reply:
Removed unused class